packages/orbit-components/config/buildExample.js

```diff
@@ -160,11 +160,16 @@ async function readFileAndCreateJSON(pathToFile) {
   const ast = parse(code, {
     sourceType: "module",
   });
+  const firstLetter = nameSplit[1].charAt(0);
+  const componentName =
+    firstLetter === firstLetter.toLowerCase() && firstLetter !== firstLetter.toUpperCase()
```

Comment:
@CollierCZ here `firstLetter === firstLetter.toLowerCase()` is enough for check :)

Reply:
I thought so too, but someone recommended both. I'll put it the simpler way. 🙂